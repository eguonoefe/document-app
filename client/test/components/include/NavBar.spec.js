/* global expect jest test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import NavBar from '../../../components/include/NavBar';

configure({ adapter: new Adapter() });

jest.dontMock('../../../components/include/NavBar');

describe('NavBar Component', () => {
  test('should match the navbar snapshot', () => {
    const match = {
      url: '/'
    };
    const onClick = jest.fn();
    const component = shallow(<NavBar match={match} onClick={onClick} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
