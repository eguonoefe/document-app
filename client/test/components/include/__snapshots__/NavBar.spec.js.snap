// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Component should match the navbar snapshot 1`] = `
<nav
  className="nav-extended nav-design"
>
  <div
    className="nav-wrapper"
  >
    <Link
      className="brand-logo"
      to="/"
    >
      <img
        alt="nav-logo"
        className="brand-img"
        src="../../img/logo2.jpg"
      />
       ms
    </Link>
    <Link
      className="button-collapse"
      data-activates="mobile-demo"
      to=""
    >
      <i
        className="material-icons"
      >
        menu
      </i>
    </Link>
    <ul
      className="right hide-on-med-and-down"
      id="nav-mobile"
    >
      <li
        className="signin-button"
      >
        <Link
          to="/auth/signin"
        >
          LOG IN
        </Link>
      </li>
      <li
        className="signup-button"
      >
        <Link
          to="/auth/signup"
        >
          GET STARTED
        </Link>
      </li>
    </ul>
    <ul
      className="side-nav"
      id="mobile-demo"
    >
      <li>
        <Link
          to="/auth/signin"
        >
          LOG IN
        </Link>
      </li>
      <li>
        <Link
          to="/auth/signup"
        >
          GET STARTED
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;
